App/src/main/java/com/gluonhq/chat/model/ChatMessage.java

```diff
@@ -10,12 +10,18 @@
     String message;
     LocalDateTime time;
     User user;
+    private boolean localOriginated;
```

Comment:
Since non of other parameters have `private` on them, may be skip it for the new parameter as well?

Reply:
+1